R/s3.get.R

```diff
@@ -28,9 +28,12 @@ s3.get <- function (path, bucket.location = "US", verbose = FALSE, debug = FALSE
     }
 
     ## Run the s3cmd tool to fetch the file from S3.
-    s3.cmd <- paste("get", paste0('"', path, '"'), x.serialized, paste("--bucket-location",
-      bucket.location), ifelse(verbose, "--verbose --progress",
-      "--no-progress"), ifelse(debug, "--debug", ""))
+    s3.cmd <- paste("get", paste0('"', path, '"'), x.serialized,
+                    bucket_location_to_flag(s3cmd(), bucket.location),
+                    ifelse(verbose,
+                           "--verbose --progress",
+                           "--no-progress"),
```

Comment:
`ifelse` isn't really meant as a ternary, it's meant as a vectorized case statement for length >1 vectors.